src/opensearch.ts

```diff
@@ -0,0 +1,113 @@
+import { ApiResponse, Client } from '@opensearch-project/opensearch'
+import { TransportRequestPromise } from '@opensearch-project/opensearch/lib/Transport'
+import { taskEither } from 'fp-ts'
+import { pipe } from 'fp-ts/lib/pipeable'
+import { chain, TaskEither, tryCatch } from 'fp-ts/lib/TaskEither'
+import { extractSource } from './extract'
+import { ApplicationError, StatusCodes } from './http'
+
+enum OSIndex {
+    Snapshots = 'snapshots_kris_1'
+}
+
+export const getClient = (): Client => {
+    return new Client({ node: 'http://localhost:9501' })
+}
+
+export const getNestedObject = (
+    path: string,
+    term: string,
+    value: string,
+    includes: string[]
+) => {
+    return {
+        path: path,
+        query: {
+            bool: {
+                filter: {
+                    term: {
+                        [path + '.' + term]: {
+                            value
+                        }
+                    }
+                }
+            }
+        },
+        ignore_unmapped: false,
+        inner_hits: {
+            ignore_unmapped: false,
+            from: 0,
+            size: 7,
```

Comment:
are you sure it should be hardcoded?

Reply:
We will handle pagination in separate PR